tool/README.MD

```diff
@@ -0,0 +1,10 @@
+ ## Generate the json configuration file used by GNI tool
```

Comment:
What is the json file used for (i.e. for which aspect of the GNI tool?). In general, be as descriptive as possible in the README's.

Reply:
Added more descriptions in README